src/sentry/templates/sentry/emails/reports/new.html

```diff
@@ -0,0 +1,371 @@
+{% extends "sentry/emails/base.html" %}
```

Comment:
For my own education, what templating system is being used for this?

Reply:
@armenzg Just Djagno: https://docs.djangoproject.com/en/2.2/topics/templates/